src/main/kotlin/nl/info/client/zgw/shared/ZGWApiService.kt

```diff
@@ -310,8 +310,12 @@ class ZGWApiService @Inject constructor(
     }
 
     private fun createStatusForZaak(zaakURI: URI, statustypeURI: URI, toelichting: String?): Status {
-        val status = Status(zaakURI, statustypeURI, ZonedDateTime.now())
-        status.statustoelichting = toelichting
+        val status = Status().apply {
+            zaak = zaakURI
+            statustype = statustypeURI
+            datumStatusGezet = ZonedDateTime.now().toOffsetDateTime()
```

Comment:
Does the `ZonedDateTime.now()` use the default TZ and therefore be the same as the `LocalDateTime.now()`, but with explicit TZ?

Reply:
In the old manual `Status` class the following was used:

```
    @JsonbDateFormat(value = DATE_TIME_FORMAT)
    private final ZonedDateTime datumStatusGezet;
 ```
 
 So quite different from the generated code in this PR:
 
 ```
   @JsonbProperty("datumStatusGezet")
  protected OffsetDateTime datumStatusGezet;
 ```   
 
 You might be on to something here.. ;-) Our itests do succeed but that does not say much in this case maybe.

Reply:
Then again, the AI says the new situation is actually more correct:

```
In the OpenAPI specification, the date-time format is used to represent a date and time as defined by RFC3339, which includes both the date and time in a combined format, such as "2020-02-12T07:20:50.52Z".

When generating Java code from an OpenAPI specification, the date-time format is typically mapped to the OffsetDateTime class in Java, which accurately represents a date and time with an offset from UTC.
```

Reply:
OpenZaak / ZGW API uses ZTC date times.